README.md

```diff
@@ -26,18 +27,19 @@ This build plugin implements Angular Support on Netlify.
 
 ## Installation and Configuration
 
-Netlify automatically detects Angular projects and sets up the latest version of this plugin. There's no further configuration needed from Netlify users.
+Netlify automatically detects Angular projects and sets up the latest version of this plugin.
 
-### Manual Installation
+### For Angular 17 and Angular 18
 
-If you need to pin down this plugin to a fixed version, install it manually.
+There's no further configuration needed from Netlify users.
 
-Create a `netlify.toml` in the root of your project. Your file should include the plugins section below:
+### For Angular 19
 
-```toml
-[[plugins]]
-  package = "@netlify/angular-runtime"
-```
+If you are using Server-Side Rendering you will need to install Angular Runtime in your Angular project to be able to import required utilities to successfully deploy request handler to Netlify. See [Manual Installation](#manual-installation) for installations details. See [Request handling](#request-handling) for more information about request handler.
+
+### Manual Installation
+
+If you need to pin down this plugin to a fixed version or you are using Server-Side Rendering with Angular 19, install it manually.
```

Comment:
If you need to pin ~down~ this plugin to a ~fixed~ **specific** version or **if** you are using Server-Side Rendering with Angular 19, **you will need to** install ~it~ **the plugin** manually.

Reply:
applied in https://github.com/netlify/angular-runtime/pull/223/commits/b086bb61fd0832909478a498d7cd61f067ecd569